pkg/sdn/plugin/common.go

```diff
@@ -14,14 +14,12 @@ import (
 	kapi "k8s.io/kubernetes/pkg/api"
 	kcache "k8s.io/kubernetes/pkg/client/cache"
 	"k8s.io/kubernetes/pkg/fields"
+	kcontainer "k8s.io/kubernetes/pkg/kubelet/container"
 )
 
 func getPodContainerID(pod *kapi.Pod) string {
 	if len(pod.Status.ContainerStatuses) > 0 {
-		// Extract only container ID, pod.Status.ContainerStatuses[0].ContainerID is of the format: docker://<containerID>
```

Comment:
I think this bit is acutally also fixed by https://github.com/openshift/origin/pull/11613